package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "waterdataui-assets",
+  "version": "1.0.0",
+  "description": "Compiled static assets for Water Data For The Nation",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "clean": "rm -rf dist && mkdir dist",
+    "build": "npm run clean && run-p build:*",
+    "build:css": "node-sass --include-path node_modules/uswds/src/stylesheets assets/styles/main.scss | uglifycss > dist/main.css",
+    "build:fonts": "mkdir -p dist/fonts && cp node_modules/uswds/src/fonts/* dist/fonts",
+    "build:images": "mkdir -p dist/img && cp -r node_modules/uswds/src/img/* dist/img",
+    "build:js": "browserify -t uglifyify assets/scripts/index.js | uglifyjs -c > dist/bundle.js",
+    "serve:static": "live-server --port=9000 --cors --no-browser dist",
+    "watch": "run-p watch:* serve:static",
+    "watch:css": "nodemon -w assets/styles -e scss -x \"npm run build:css\"",
+    "watch:fonts": "nodemon -w node_modules/uswds/src/fonts -x \"npm run build:fonts\"",
+    "watch:images": "npm run build:images",
+    "watch:js": "watchify assets/scripts/index.js -o dist/bundle.js -dv",
+    "watch:python": "./run.py"
```

Comment:
I noticed that everything is set to run on port 9000. Perhaps this is ignorance on my part, but should `app.run(host=host, port=5050, threaded=True)` in run.py be changed to `app.run(host=host, port=9000, threaded=True)`, or does it not matter?

Reply:
It can stay at localhost:5050 - the Flask hosted app load static files from localhost:9000